openssl/src/ssl/mod.rs

```diff
@@ -339,7 +373,13 @@ impl SslContext {
             return Err(SslError::get());
         }
 
-        Ok(SslContext { ctx: ctx })
+        let ctx = SslContext { ctx: ctx };
+
+        if method.is_dtls() {
+            ctx.set_read_ahead(1);
```

Comment:
The openssl man pages seem to imply that this has no effect if DTLS is being used - am I misinterpreting that? http://wiki.openssl.org/index.php/Manual:SSL_CTX_set_read_ahead%283%29